stock_archive_constraint/tests/test_location_archive_constraint.py

```diff
@@ -60,7 +60,6 @@ def _create_stock_move_line(self, location_id, location_dest_id, product_id, qty
         stock_move_line_form.location_id = location_id
         stock_move_line_form.location_dest_id = location_dest_id
         stock_move_line_form.product_id = product_id
-        stock_move_line_form.product_uom_qty = qty
```

Comment:
I think it should be:
```python
stock_move_line_form.product_qty = qty
```
although it can be the same without assigning such value.

Reply:
https://github.com/OCA/OCB/blob/52901f6aa97a47e8cef89695fcbab1fbaee1081d/addons/stock/models/stock_move_line.py#L77-L82

and this field is not defined in the picking view:
https://github.com/OCA/OCB/blob/52901f6aa97a47e8cef89695fcbab1fbaee1081d/addons/stock/views/stock_picking_views.xml#L282-L299

Reply:
OK then